site/app/templates/Vue.twig

```diff
@@ -0,0 +1,8 @@
+<script> 
+    window.addEventListener('load', () => {
+        window.submitty.render('[data-vuemount]', "{{ page }}",  {{ args | json_encode() | raw() }});
```

Comment:
It would be better to use the `js` filter instead of `raw` to prevent XSS attacks.

Reply:
I can't find this `js` filter, as Twig says it doesn't exist and it does not appear on the [documentation website](https://twig.symfony.com/doc/3.x/). If you meant to say to use the `escape` filter with the argument `'js'` (`e('js')`), that would defeat the entire point of why I'm using `raw`. In order for this to work, I need to pass a javascript object to `window.submitty.render`, and using the `e('js')` filter breaks that. I'm currently unaware of another way of doing this.
If it helps, I believe XSS attacks are impossible here anyway due to `args` being a trusted parameter that only the backend can provide.

Reply:
I forgot that it's not a js filter, but rather an escape filter with js as an option.  Apologies for the confusion.  `... | json_encode | e('js')` should work here though, right?

While there are no *current* XSS issues here, we want to minimize the usage of the `raw` filter throughout the codebase if at all possible, because someone could come along later and accidentally open up something when they don't see the full picture about what's going on.  It's happened before...

Reply:
>I need to pass a javascript object to `window.submitty.render`, and using the `e('js')` filter breaks that.

Using `raw`:
```js
$('document').ready(() => {
    window.submitty.render('[data-vuemount]', "Example",  {"example":"value"});
})
```
Using `e('js')`:
```js
$('document').ready(() => {
    window.submitty.render('[data-vuemount]', "Example", \u007B\u0022example\u0022\u003A\u0022value\u0022\u007D);
})
```

I can attempt to find some other way to pass this data to Vue, or we could just add a comment explaining why it needs to be used here and how exactly it should be used. What would you prefer?

Reply:
Alright, that makes sense.  I think it would be fine to stick with `raw` for now then.  Perhaps in the future we could create a `json` filter which handles this via some common code path.